### DISCUSSION TITLE
Allow soundfont patches to be seen in Rack mode drop downs similar to plugin presets

### DISCUSSION THREAD

**Discussion by audio2000** [falkTX/Carla#1914] [Ideas]
Carla has the ability to load up soundfonts which load up as modules.
When in Rack mode, maybe can consider being able to add drop downs similar to how plugin presets are accessed in Rack modules.
This would allow user access to patches within soundfonts that contain more than one patch.
Would have to decide how soundfont banks would be displayed / accesssed.
Not sure if anyone else would find this useful.

Thank You

**Comment by jpka-** [comment]
Hello. Is _Change Skin_ to `mpresets`, is what you mean?

![mpresets](https://github.com/user-attachments/assets/6159f050-5ef0-4472-8d02-7f5277be697d)
